handsontable/src/plugins/filters/filters.js

```diff
@@ -432,11 +422,22 @@ export class Filters extends BasePlugin {
   /**
    * Gets last selected column index.
    *
-   * @returns {object|null} Return `null` when column isn't selected otherwise
-   * object containing information about selected column with keys `visualIndex` and `physicalIndex`.
+   * @returns {{visualIndex: number | null, physicalIndex: number | null}} Return `null` when column isn't selected
```

Comment:
@budnix Can you please check, the above line is serialized to Markdown as: 

```
**Returns**: `Object` - Return `null` when column isn't selected
otherwise object containing information about selected column with keys `visualIndex` and `physicalIndex`.  
```

I mean that there is just `Object` instead of the complex type that you've entered.

Reply:
It seems that our JSDoc compiler is not able to generate such a precise type so it widening to `Object` type.

Reply:
Correct, but fully informative for someone who reads the API reference https://handsontable.com/docs/api/filters/.

We should at least fix the description text of the return value, which still says:

> Return `null` when column isn't selected otherwise object containing information about selected column with keys `visualIndex` and `physicalIndex`

Could you please propose a new description text for the return value and I will put it to https://github.com/handsontable/handsontable/pull/9913

Reply:
Maybe something like [this](https://github.com/handsontable/handsontable/commit/bbf09f92258a9ca69e935cc4426625ae89cfadd9#diff-01aa731e6c00facb0cd36cd88d50dcd8a9e90437847f80fd5db3bddb1880ccd6R425)?

Reply:
Looking good, thanks! I cherry-picked it to PR #9913